EmotiBit.cpp

```
@@ -3763,7 +3856,29 @@ void EmotiBit::setPowerMode(PowerMode mode)
 	else if (getPowerMode() == PowerMode::WIRELESS_OFF)
 	{
 		Serial.println("PowerMode::WIRELESS_OFF");
+
+		if (_enableBluetooth == true)
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)